webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {Button} from 'react-bootstrap';
+import Select from '../../../../../common/forms/Select';
+import Checkbox from '../../../../../common/forms/Checkbox';
+import TextInput from '../../../../../common/forms/TextInput';
+import './disk.scss';
+
+const Disk = (
+  {
+    removeDisk,
+    updateDisk,
+    name,
+    config: {
+      datastores,
+      storagePods,
+      diskModeTypes
+    },
+    storagePod,
+    dataStore,
+    size,
+    thinProvision,
+    eagerZero,
+    mode
+  }
+) => {
+  return (
+    <div className="disk-container">
+      <div className="form-group">
+        <label className="col-md-2 control-label">
+          {__('Disk name')}
+        </label>
+        <div className="col-md-4">
+          {name}
+        </div>
+      </div>
+      {!dataStore &&
+        <Select
+          label={__('Storage Pod')}
+          value={storagePod}
+          onChange={updateDisk.bind(this, 'storagePod')}
+          options={storagePods}
+        />}
+      {!storagePod &&
+        <Select
+          label={__('Data store')}
+          value={dataStore}
```

Comment:
When editing a host, the current value is not picked up. E.g. the select field shows "Please select".